Resources/Prototypes/Roles/Jobs/Cargo/salvage_specialist.yml

```diff
@@ -3,7 +3,7 @@
   name: job-name-salvagespec
   description: job-description-salvagespec
   playTimeTracker: JobSalvageSpecialist
-  antagAdvantage: 3 # DeltaV - Reduced TC: External Access + Free hardsuit and weapons
+  antagAdvantage: 2 # DeltaV - Reduced TC: Free hardsuit and weapons
```

Comment:
salv are almost expected to have smgs and shotguns on their back at all times, insuls and goggles, can get ultra robust armour on expeds

why would you reduce difficulty even further its probably easier than traitor captain

Reply:
That's fair, would you say just leave it at 3?

Reply:
3 maybe 4

Reply:
I'll see if anyone else has thoughts so I'm not making 2 billion commits tweaking it

Reply:
Reverted it to 3